notebooks/pluto/VariogramModeling.jl

```diff
@@ -0,0 +1,184 @@
+
+# ╔═╡ 206af8a0-184a-11eb-2223-616291fcd27e
+𝒟 = geostatsimage("WalkerLake")
+
+# ╔═╡ 241f9c30-184a-11eb-04ef-2139b4670992
+plot(𝒟)
+
+# ╔═╡ f90ceec2-1840-11eb-02e8-777f5d33918f
+md"""We sample a thousand points from the image and plot them:"""
+
+# ╔═╡ 13459440-1841-11eb-3f1b-ef74d30c10b2
+begin
+	𝒮 = sample(𝒟, 1000)
+	plot(𝒮)
+end
+
+# ╔═╡ 1d0fd5d0-1841-11eb-271f-ab7825a69fec
+md"""## Empirical variogram
+
+We consider a simple omnidirectional variogram. Variogram objects in GeoStats.jl are integrated with the Julia Plots.jl framework. We can plot them directly and interact with the result as we vary the number of lags used for estimation:"""
+
+# ╔═╡ 2480c170-186a-11eb-021f-b7a99755a0d4
+md"nlags = $(@bind nlags Slider(1:30; default=15, show_value=true))"
+
+# ╔═╡ c0c8ad50-1841-11eb-18f7-37e327fd0ae5
+plot(EmpiricalVariogram(𝒮, :Z, nlags=nlags, maxlag=200.))
+
+# ╔═╡ c74aefd0-1841-11eb-1865-dfaae510218d
+md"""Besides the variogram itself, GeoStats.jl presents the bin counts (scaled) as a measure of confidence about the estimated points. This frequency plot can be deactived by passing the option `showbins=false` to the plot command. We encourage users to keep the bin counts option activated as it has zero cost.
+
+This empirical variogram was constructed using the Euclidean distance between data locations. We can also specify a custom distance to estimate the variogram when points are embedded on different coordinate systems. Please consult the documentation for more distance functions.
+
+After interacting with the plot, we select a number of lags and proceed to fitting a theoretical model:"""
+
+# ╔═╡ d35b50d0-1841-11eb-0814-676fc99dc20d
+γₑ = EmpiricalVariogram(𝒮, :Z, nlags=17, maxlag=200.)
+
+# ╔═╡ d8ab2dd0-1841-11eb-3116-0fd157960d7e
+plot(γₑ)
+
+# ╔═╡ dd031640-1841-11eb-2fb7-f750c7369758
+md"""## Theoretical variogram
+
+We will exploit the Interact.jl package again to fit a theoretical variogram to the data. Various theoretical variogram models are available in GeoStats.jl, including a composite additive model that can be used to combine different variogram types. Please consult the documentation for more details. Here we will use a simple spherical variogram:"""
+
+# ╔═╡ 8be1d270-1868-11eb-09ec-697b71485822
+md"s = $(@bind s Slider(range(0, stop=.1, length=50); default=0.05, show_value=true))"
+
+# ╔═╡ 50706cf0-1869-11eb-375f-9112e5f435e9
+md"r = $(@bind r Slider(range(0, stop=100., length=50); default=50.0, show_value=true))"
+
+# ╔═╡ 608a9660-1869-11eb-170f-699e74bd5a47
+md"n = $(@bind n Slider(range(0, stop=.1, length=50); default=0.05, show_value=true))"
+
+# ╔═╡ 824e62e0-1869-11eb-1d9b-4b650c21564f
+begin
+	γₜ1 = SphericalVariogram(sill=Float64(s), range=Float64(r), nugget=Float64(n))
```

Comment:
It looks like we have an extra white space here. Also, did you open an issue on GitHub about the slider type instability? It is not ideal to explicitly type Float64 here.

Reply:
I think I didn't get you about _extra white space_

and Yup I have opened an issue regarding Instability on PlutoUI
Ref: https://github.com/fonsp/PlutoUI.jl/issues/64

Reply:
It seems that the first line contains more than 4 whitespaces. It may be a GitHub viewer issue though.